### PR TITLE
Getting Started with Dashboards 

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?

Adds getting started with dashboards page

### Motivation

Docs OKR/DOCS-899

### Preview link
https://docs-staging.datadoghq.com/kari/getting-started-dashboards/getting_started/dashboards/

### CODE REVIEW

File: content/en/getting_started/dashboards/_index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kinds of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
+
+Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those conversations into the same dashboard. Create separate dashboards to show answers to different kinds of questions.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:23]:
```suggestion
Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those solutions into the same dashboard. Create separate dashboards to show answers to different kinds of questions.
```

File: content/en/getting_started/dashboards/_index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  tag: "Documentation"
+  text: "Tell a story about data with Notebooks"
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kinds of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution?
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:21]:
```suggestion
The key to getting started with dashboards is knowing what kind of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
```

File: content/en/getting_started/dashboards/_index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kinds of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
+
+Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those conversations into the same dashboard. Create separate dashboards to show answers to different kinds of questions.
+
+This guide gets you started on a path to making dashboards. These basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:25]:
```suggestion
This guide gets you started on a path to creating dashboards. The basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
```

File: content/en/getting_started/dashboards/_index.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## Prerequisites
+
+If you haven't already, create a [Datadog account][1]. Install an agent on a host, and an integration for something running on that host.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:29]:
```suggestion
If you haven't already, create a [Datadog account][1]. Install the Agent on a host, and an integration for something running on that host.
```

Comment (sarina-dd) [reply content/en/getting_started/dashboards/_index.md:29]: Maybe some links to the agent setup / integrations page here, just to make it easier for people to get up and running.

File: content/en/getting_started/dashboards/_index.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+## Plan
+
+Decide if your dashboard will be a timeboard (all graphs on the same time scale) or screenboard (wider variety of widgets, different time scales). Read more about [timeboards and screenboards][2] if you're not sure.
+
+Also think about what purpose you're creating the dashboard for. A dashboard can help you and your teammates focus on the right work. A _team dashboard_ reminds you what's high priority, what needs attention now, and what you're succeeding at. Make a team dashboard (or multiple) with the information that people most frequently need they have to dig for. SLO and SLI details make for an excellent team dashboard.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:35]:
```suggestion
Also, determine the purpose of the dashboard you're creating. A dashboard can help you and your teammates focus on the right work. A _team dashboard_ reminds you what's high priority, what needs attention now, and what you're succeeding at. Make a team dashboard (or multiple) with the information that people most frequently need they have to dig for. SLO and SLI details make for an excellent team dashboard.
```

Comment (sarina-dd) [reply content/en/getting_started/dashboards/_index.md:35]: https://docs.datadoghq.com/dashboards/widgets/slo/ - would be good to link to the SLO widget here.

File: content/en/getting_started/dashboards/_index.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+A dashboard connected to real-time data is a powerful tool for guiding conversations with managers and executives. A good _executive dashboard_ can show that you are working on the most important things, how much a service you use is costing or helping, and whether you're progressing toward goals, meeting your SLOs, and scaling effectively. Executive dashboards are most effective when they answer these questions at the highest level and are interconnected to enable drilling down.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:37]:
```suggestion
A dashboard connected to real-time data is a powerful tool for guiding conversations with managers and executives. A good _executive dashboard_ can show that you are working on the most important things, how much a service is costing you, or whether you're progressing toward goals, meeting your SLOs, and scaling effectively. Executive dashboards are most effective when they answer these questions at the highest level and are interconnected to drill down into the answer.
```

File: content/en/getting_started/dashboards/_index.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+Dashboards can also help you track down persistent problems and fix them. _Troubleshooting dashboards_ often start as a scratch pad of things you know, and gradually build as you discover more. For example, start with a graph or widget from another dashboard or view that shows a problem-you'll be drilling down from there to find your solution.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:39]:
```suggestion
Dashboards can also help you track down persistent problems and fix them. _Troubleshooting dashboards_ often start as a scratch pad of things you know, and gradually build as you discover more. For example, start with a graph or widget from another dashboard or view that shows a problem. You can then drill down from there to find your solution.
```

File: content/en/getting_started/dashboards/_index.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+## Explore out-of-the-box dashboards
+
+Datadog provides many out-of-the-box dashboards for features and integrations. For the infrastructure you monitor, check out the out-of-the-box dashboards that are provided with Datadog:
+
+1. Go to **Dashboards > Dashboards list** and search for the name of an integration you have added, for example `Redis`, or a feature you use, such as `RUM`. 
+2. Browse the search results for dashboards marked *Preset* and see if at least some of the graphs show the answers you're looking for.
+3. Explore the links in the out-of-the-box dashboard's title drop-down to find more information about how people are using them.
+
+## Start by reusing good things
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:49]:
```suggestion
## Start by reusing other dashboards
```

File: content/en/getting_started/dashboards/_index.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+A common way to start a dashboard is by encountering a similar dashboard already in use, and adjusting it to suit your needs. If you find a dashboard that answers many of the questions you want your dashboard to answer: 
+
+1. Clone it by opening the dashboard and selecting **Clone dashboard** from the Settings menu (the gear icon on the right-hand side). This creates an unlinked copy of the dashboard; changes you make in the new copy don't affect the source widget.
+2. Edit the clone by opening it and clicking **Edit widgets**. 
+3. Delete widgets you don't use by selecting **Delete** from the widget's Settings menu.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:55]:
```suggestion
3. Delete widgets you don't need by selecting **Delete** from the widget's Settings menu.
```

File: content/en/getting_started/dashboards/_index.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+
+1. Go to **Dashboards > Dashboards list** and search for the name of an integration you have added, for example `Redis`, or a feature you use, such as `RUM`. 
+2. Browse the search results for dashboards marked *Preset* and see if at least some of the graphs show the answers you're looking for.
+3. Explore the links in the out-of-the-box dashboard's title drop-down to find more information about how people are using them.
+
+## Start by reusing good things
+
+A common way to start a dashboard is by encountering a similar dashboard already in use, and adjusting it to suit your needs. If you find a dashboard that answers many of the questions you want your dashboard to answer: 
+
+1. Clone it by opening the dashboard and selecting **Clone dashboard** from the Settings menu (the gear icon on the right-hand side). This creates an unlinked copy of the dashboard; changes you make in the new copy don't affect the source widget.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:53]: Might be good to throw in a screenshot or gif in to show how to clone

File: content/en/getting_started/dashboards/_index.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+2. Edit the clone by opening it and clicking **Edit widgets**. 
+3. Delete widgets you don't use by selecting **Delete** from the widget's Settings menu.
+4. Move things around to suit your needs. Groups and individual widgets can be dragged and dropped into new locations in the dashboard.
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
+5. Use the **Export to Dashboard** option provided by many Datadog views for data they show. For example, the Logs Explorer and Log Analytics views have share options to export logs lists and metrics to dashboards.
+
+## Figuring out what a metric means
+
+Through integrations, Datadog collects [metrics][3] about your infrastructure and applications. The collected metrics are documented in the integration's Readme files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric means, look it up in the Integrations docs.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:62]:
```suggestion
Through integrations, Datadog collects [metrics][3] from your infrastructure and applications. The collected metrics are documented in the integration's README files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric is, look it up in the Integrations docs. 
```

Comment (sarina-dd) [reply content/en/getting_started/dashboards/_index.md:62]: Probably helpful to drop a link to an existing integrations doc with metrics for context

File: content/en/getting_started/dashboards/_index.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+4. Move things around to suit your needs. Groups and individual widgets can be dragged and dropped into new locations in the dashboard.
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
+5. Use the **Export to Dashboard** option provided by many Datadog views for data they show. For example, the Logs Explorer and Log Analytics views have share options to export logs lists and metrics to dashboards.
+
+## Figuring out what a metric means
+
+Through integrations, Datadog collects [metrics][3] about your infrastructure and applications. The collected metrics are documented in the integration's Readme files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric means, look it up in the Integrations docs. 
+
+For example, suppose you are looking at a time graph of the metric `aws.s3.first_byte_latency`. Go to the [Data collected][5] section of the AWS S3 integration Readme to see its description:
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:64]:
```suggestion
For example, suppose you are looking at a time graph of the metric `aws.s3.first_byte_latency`. Go to the [Data collected][5] section of the AWS S3 integration README to see its description: 
```

File: content/en/getting_started/dashboards/_index.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
+5. Use the **Export to Dashboard** option provided by many Datadog views for data they show. For example, the Logs Explorer and Log Analytics views have share options to export logs lists and metrics to dashboards.
+
+## Figuring out what a metric means
+
+Through integrations, Datadog collects [metrics][3] about your infrastructure and applications. The collected metrics are documented in the integration's Readme files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric means, look it up in the Integrations docs. 
+
+For example, suppose you are looking at a time graph of the metric `aws.s3.first_byte_latency`. Go to the [Data collected][5] section of the AWS S3 integration Readme to see its description: 
+
+    The average per-request time from the complete request being received by a bucket to when the response starts to be returned. Shown as millisecond
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:66]:
```suggestion
    `The average per-request time from the complete request being received by a bucket to when the response starts to be returned. Shown as millisecond.`
```

File: content/en/getting_started/dashboards/_index.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+
+## Try something not strictly timeseries metrics
+
+Timeseries graphs of metrics are useful, but dashboards can contain many types of widgets to communicate important information. Try:
+
+ - **Alert values and Check statuses**: Show big red, yellow, and green numbers to bring attention to successes or problems.
+ - **Heat maps**: Show complex metric-infrastructure relationships across multiple tags with intuitive color-intensity graphs.
+ - **iFrames, formatted text, and images**: Show any number of website-like details to help explain the dashboard contents and provide additional resources.
+ - **Tables**: Show lists of metrics grouped by tag keys.
+ - **Top lists**: Show, for example, which hosts have the least free space, or which services are throwing the most errors or which URLs are returning the most 404s.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:86]:
```suggestion
 - **Top lists**: For example, show which hosts have the least free space, which services are throwing the most errors, or which URLs are returning the most 404s.
```

**Review by sarina-dd — CHANGES_REQUESTED** [review]
👍 Looks great so far! Only overall suggestion is a few screenshots or gifs to help guide/give users an idea of what they're looking for/at.

File: content/en/getting_started/dashboards/_index.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+  text: "Monitors, SLOs, notifications, downtimes, and incidents"
+---
+
+## Overview
+
+The key to getting started with dashboards is knowing what kind of questions you ask yourself regularly. What are common issues your customers face? When a problem occurs, what questions help you find a solution? 
+
+Creating a good dashboard is about bringing the answers to these questions to the surface. Also, it is important not to cram all of those thoughts into the same dashboard. Creating separate dashboards to pinpoint different issues can help you quickly find your answers.
+
+This guide gets you started on a path to creating dashboards. The basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:25]:
```suggestion
This guide gets you started on a path to creating dashboards. These basic dashboards will help your team talk about things that really matter and get to the bottom of problems quickly.
```

File: content/en/getting_started/dashboards/_index.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+## Start by reusing other dashboards
+
+A common way to start a dashboard is by encountering a similar dashboard already in use, and adjusting it to suit your needs. If you find a dashboard that answers many of the questions you want your dashboard to answer: 
+
+1. Clone it by opening the dashboard and selecting **Clone dashboard** from the Settings menu (the gear icon on the right-hand side). This creates an unlinked copy of the dashboard; changes you make in the new copy don't affect the source widget.
+    {{< img src="getting_started/dashboards/cloning_dashboard.gif" alt="Cloning a dashboard"  >}}
+2. Edit the clone by opening it and clicking **Edit widgets**. 
+3. Delete widgets you don't need by selecting **Delete** from the widget's Settings menu.
+4. Move things around to suit your needs. Groups and individual widgets can be dragged and dropped into new locations in the dashboard.
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:58]:
```suggestion
5. Copy in widgets you like from other dashboards by hovering over the widget and typing `Command + C` (`Ctrl + C` on Windows). Paste it into your dashboard by opening the dashboard and typing `Command + V` (`Ctrl + V` on Windows).
```

File: content/en/getting_started/dashboards/_index.md [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+
+There are two ways to create links from a dashboard to any target URL:
+
+ - Add a Notes and Links widget, which can contain Markdown formatted text, including links. The widget editor includes Markdown formatting tips.
+ - Create a Custom link from a widget's Settings (gear) menu. Custom links can interpolate variables and template variables, so that the link changes according to what the user has selected when they click, taking them to exactly the right place for drilling down into data or taking corrective action. 
+     {{< img src="getting_started/dashboards/opening_custom_link.gif" alt="Opening a custom link"  >}}
+
+## What's next 
+
+### Share your dashboards outside the Datadog app
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:105]:
```suggestion
### Share your dashboards outside of the Datadog app
```

File: content/en/getting_started/dashboards/_index.md [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+
+After you've selected a few metrics to add to your dashboard, experiment with various [widget types][6], [queries][7], [functions][8], and [aggregation approaches][9], to display the data in ways that best answer the questions you have. 
+
+By specifying Template variables, you can make one dashboard answer questions for a selection of scenarios. For example, you can create a time graph that shows latency metrics for whichever data center geography the user selects from the dashboard's variables drop-down, or for all of them together. For more information, see [Template Variables][10].
+
+You can make graphs easier to read by adjusting Y-axis ranges, colors, or legends, or by adding markers and event overlays. See the [Dashboards documentation][11] for all the ways you can customize and refine [timeseries][12] and [other widgets][6].
+
+For more details and examples of these techniques, sign up for the online learning course [Building Better Dashboards][13].
+
+## Try something not strictly timeseries metrics
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:77]:
```suggestion
## Try out other widgets
```

File: content/en/getting_started/dashboards/_index.md [hunk 19]
```diff
@@ -0,0 +1,135 @@
...
+
+1. Clone it by opening the dashboard and selecting **Clone dashboard** from the Settings menu (the gear icon on the right-hand side). This creates an unlinked copy of the dashboard; changes you make in the new copy don't affect the source widget.
+    {{< img src="getting_started/dashboards/cloning_dashboard.gif" alt="Cloning a dashboard"  >}}
+2. Edit the clone by opening it and clicking **Edit widgets**. 
+3. Delete widgets you don't need by selecting **Delete** from the widget's Settings menu.
+4. Move things around to suit your needs. Groups and individual widgets can be dragged and dropped into new locations in the dashboard.
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
+5. Use the **Export to Dashboard** option provided by many Datadog views for data they show. For example, the Logs Explorer and Log Analytics views have share options to export logs lists and metrics to dashboards.
+
+## Figuring out what a metric means
```
Comment (sarina-dd) [inline content/en/getting_started/dashboards/_index.md:61]:
```suggestion
## Learn more about metrics
```

**Review by sarina-dd — COMMENTED** [review]
Left a few small additional edits I missed in the first round and tried to reword a few headers to be active / a little shorter. Looks great! 👍

**Review by apigirl — APPROVED** [review]

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**